.github/workflows/crib-integration-test.yml

```diff
@@ -3,9 +3,13 @@ on:
   pull_request:
     paths:
       - ".github/workflows/crib-integration-test.yml"
-  schedule:
-    - cron: "0 1 * * *"
   workflow_call:
+    inputs:
```

Comment:
Please correct me if I'm wrong, but is this going to trigger the test every time we merge into the develop branch? If that's true, a quick check shows there are 10+ PRs daily, resulting in at least 300 runs per month. This could cost us a significant amount of money, I suppose.
My questions are:
- Do we want to spend money on a broken test? It will spin up some pods every time, only to fail.
- Does it really make sense to enable it for every push to develop, considering we can't make it a mandatory check and we don't expect people to fix CRIB?

Maybe it would make sense to completely disable it for now. Once it's fixed, we could run it on a schedule (e.g., once daily) or trigger it by adding a specific label to PRs.